apitools/org.eclipse.pde.api.tools.tests/src/org/eclipse/pde/api/tools/problems/tests/ApiProblemTests.java

```diff
@@ -581,20 +584,7 @@ public void testGetHashCodeResourcePathNull() {
 		IApiProblem problem = ApiProblemFactory.newApiProblem(null, null, new String[] { "test1, test2, test3" }, null, null, 2, 2, 2, IApiProblem.CATEGORY_COMPATIBILITY, IElementDescriptor.FIELD, IApiProblem.ILLEGAL_IMPLEMENT, IDelta.ANNOTATION_DEFAULT_VALUE); //$NON-NLS-1$
 		assertNotNull("there should have been a new problem created", problem); //$NON-NLS-1$
 		assertEquals("the hashcode should be equal to the sum of: id, resourcepath.hashCode", //$NON-NLS-1$
-				problem.hashCode(), (problem.getId() + 0 + argumentsHashcode(new String[] { "test1, test2, test3" }))); //$NON-NLS-1$
+				problem.hashCode(), (problem.getId() + 0 + Arrays.hashCode(new String[] { "test1, test2, test3" }))); //$NON-NLS-1$
```

Comment:
`Objects.hash()` uses var-args and you could save here and above the explicit creation of an array.
```suggestion
				problem.hashCode(), (problem.getId() + 0 + Objects.hash("test1, test2, test3"))); //$NON-NLS-1$
```

Reply:
nice